frontend/src/components/dialog/DialogBase.vue

```diff
@@ -57,10 +57,17 @@ export default {
       })
       this.embeddedEntities.forEach((key) => {
         if (data[key]) {
-          loadingPromises.push(data[key]()._meta.load)
-          data[key]()._meta.load.then((obj) =>
-            this.$set(this.entityData, key, obj._meta.self)
-          )
+          if (typeof data[key] === 'function') {
+            loadingPromises.push(data[key]()._meta.load)
+            data[key]()._meta.load.then((obj) =>
+              this.$set(this.entityData, key, obj._meta.self)
+            )
+          } else {
+            loadingPromises.push(data[key]._meta.load)
+            data[key]._meta.load.then((obj) =>
+              this.$set(this.entityData, key, obj._meta.self)
+            )
+          }
```

Comment:
You could DRY this up using:
```suggestion
          const promise = typeof data[key] === 'function' ? data[key]()._meta.load : data[key]._meta.load
          loadingPromises.push(data[key]()._meta.load)
          data[key]()._meta.load.then((obj) =>
            this.$set(this.entityData, key, obj._meta.self)
          )
```

Reply:
I assume you mean:
```
const promise = typeof data[key] === 'function' ? data[key]()._meta.load : data[key]._meta.load
loadingPromises.push(promise)
promise.then((obj) =>
   this.$set(this.entityData, key, obj._meta.self)
)
```

Will adjust accordingly

Reply:
Fixed in [5d15a23](https://github.com/ecamp/ecamp3/pull/3797/commits/5d15a234a52f1e14bd02ee313fa3b4691c4f12ae)